src/main/scala/org/broadinstitute/dsde/workbench/sam/config/KmsConfig.scala

```diff
@@ -0,0 +1,22 @@
+package org.broadinstitute.dsde.workbench.sam.config
+
+import org.broadinstitute.dsde.workbench.google.{KeyId, KeyRingId, Location}
+import org.broadinstitute.dsde.workbench.model.google.GoogleProject
+
+/**
+  * created by mtalbott 1/25/18
+  *
+  * @param project google project for the key
+  * @param location location for the key
+  * @param keyRingId name of the key ring that contains the key
+  * @param keyId name of the key
+  */
+
+final case class GoogleKmsConfig(
+                                  project: GoogleProject,
+                                  location: Location,
+                                  keyRingId: KeyRingId,
+                                  keyId: KeyId,
+                                  rotationPeriod: Int
```

Comment:
what is `Int`? if this is duration, we should probably use `FiniteDuration`

Reply:
It's the number of days. I call `.days.toSeconds` on it. I suppose we could use a Duration, but the number of days as an int was what I thought of first

Reply:
so lots config libraries support reading `FiniteDuration` directly. x days probably can be `x days` in config directly...see https://github.com/iheartradio/ficus, there's an example for `minutes`, but I think it should definitely have `day` too

```
  // something such as "15 minutes" can be converted to a FiniteDuration
  val retryInterval: FiniteDuration = config.as[FiniteDuration]("retryInterval")
```

Reply:
did a bit more digging, I see https://github.com/iheartradio/ficus/blob/a6db5d65f3c3237a27aee592661678962a2aca09/src/test/scala/net/ceedubs/ficus/readers/DurationReadersSpec.scala#L34, I think you should be able to do `180 days` in config file and read it as a `FiniteDuration`